user_manual/groupware/mail.rst

```diff
@@ -65,6 +65,23 @@ Mail can move a message to a different folder when it is marked as junk.
 
    .. figure:: images/mail_move-message-to-junk-folder.png
 
+Enable mail body search
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+   .. versionadded:: 3.5
+
+Mail bodies can now be searched, this feature is opt-in because of potential performance issues.
+
```

Comment:
I would rephrase it like this: "You now have the ability to search through email bodies. However, this feature is optional due to potential performance concerns."